tests/snapshot-gating.fmf

```diff
@@ -37,6 +27,10 @@ prepare:
       - dnf -y install --best llvm-libs
       # Use a newer llvm-test-suite version from copr.
       - dnf copr enable -y @fedora-llvm-team/llvm-test-suite $COPR_CHROOT
+      # Increase the priority of the copr repositories in order to prefer packages from them.
+      # Testing Farm uses priority=9 by default.
+      # Source: https://docs.testing-farm.io/Testing%20Farm/0.1/test-environment.html#_tag_repository
+      - dnf config-manager --save --setopt="copr:copr.fedorainfracloud.org:group_fedora-llvm-team:*.priority=8" || dnf config-manager setopt "copr:copr.fedorainfracloud.org:group_fedora-llvm-team:*.priority=8"
```

Comment:
Did you test locally whether this common works after enabling the copr repo?

Reply:
I did run manually the DNF5 command in a container configured by hand.
It worked.

I'm afraid the actual environment is far more diverse, though.